internal/command/launch/launch.go

```diff
@@ -349,13 +351,69 @@ func run(ctx context.Context) (err error) {
 	if !(srcInfo == nil || srcInfo.SkipDatabase || flag.GetBool(ctx, "no-deploy") || flag.GetBool(ctx, "now")) {
 		confirmPg, err := prompt.Confirm(ctx, "Would you like to set up a Postgresql database now?")
 		if confirmPg && err == nil {
-			LaunchPostgres(ctx, appConfig.AppName, org, region)
+			db_app_name := fmt.Sprintf("%s-db", appConfig.AppName)
+			should_attach_db := false
+
+			if apps, err := client.GetApps(ctx, nil); err == nil {
+				for _, app := range apps {
+					if app.Name == db_app_name {
+						msg := fmt.Sprintf("We found an existing Postgresql database with the name %s. Would you like to attach it to your app?", app.Name)
+						confirmAttachPg, err := prompt.Confirm(ctx, msg)
+
+						if confirmAttachPg && err == nil {
+							should_attach_db = true
+
+						}
+
+					}
+
+				}
+
+			}
+
 			options["postgresql"] = true
+
+			if should_attach_db {
+				current_time := time.Now().Nanosecond()
+				db_user := fmt.Sprintf("%s-%d", db_app_name, current_time)
+				fmt.Println(db_user)
```

Comment:
How about removing this, or adding context?

Reply:
Definitely should add context. If we just try to use the typical username generation format of using the app's name ,when we'll run into the issue of not having a unique username. By using a sufficiently random number (in this case, the nano second that this command is being run at), we can pretty much guarantee we're creating a unique user. I'll add a comment about this.